packages/headless/src/chat.ts

```diff
@@ -261,6 +271,13 @@ export function useChat(onMessage?: OnMessage, userId?: number) {
     [socket]
   );
 
+  const userTypingMessage = useCallback(
```

Comment:
The name of the function should start with a **verb**. You can prefix it with `ack` which a common abbreviation for `acknowledge` (the same as emit)


```suggestion
  const ackUserTyping = useCallback(
```